packages/zapp/console/src/components/Launch/LaunchForm/ResumeFormHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import { DialogTitle, Typography } from '@material-ui/core';
+import * as React from 'react';
+import { formStrings } from './constants';
+import { useStyles } from './styles';
+
+export const ResumeFormHeader: React.FC<{ title?: string }> = ({ title = '' }) => {
+  const styles = useStyles();
+  return (
+    <DialogTitle disableTypography={true} className={styles.header}>
+      <div className={styles.inputLabel}>{formStrings.resumeTitle}</div>
+      <Typography variant="h6">{title}</Typography>
+    </DialogTitle>
+  );
+};
```

Comment:
This is 99% duplicate of `LaunchFormHeader`, we need to reuse the existing component and just pass down appropriate `formTitle` prop based on the place it's being rendered (formStrings.resumeTitle or formStrings.title)